src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  --link-color: #00ACDC;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content:space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+nav ul{
+  display: flex;
+  align-items: center;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)